Auditing for OSS documentation (#7985)
treeverse/lakeFS: docs/enterprise/orchestration.md

The distinction between regular logs and audit_logs is in the boolean field log_audit.
lakeFS and fluffy share the same configuration structure under logging.* section in the config.

{: .warning }

> > The log entries marked with log_audit = true are currently available in the open-source version of lakeFS. However, please be aware that these log entries are intended to be deprecated in future releases.

Reviewer: ```suggestion
> The log entries marked with `log_audit = true` are currently available in both the open-source version of lakeFS and lakeFS Enterprise. However, please be aware that these log entries are deprecated in the open source version and will be removed in future releases.
```
